<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publicTransportConfig</class>
 <widget class="QWidget" name="publicTransportConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>275</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data source</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Service provider:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>64</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Appearance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="KComboBox" name="serviceProvider"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="KLineEdit" name="city"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblCity">
     <property name="text">
      <string>City:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="26" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Update automatically:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QCheckBox" name="updateAutomatically">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <layout class="QHBoxLayout" name="layoutUpdateTimeout">
     <item>
      <widget class="KIntSpinBox" name="updateTimeout">
       <property name="minimum">
        <number>30</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Update every:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QCheckBox" name="showRemainingMinutes">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="2">
    <widget class="QCheckBox" name="showDepartureTime">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Show remaining minutes:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Show departure time:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="KLineEdit" name="stop"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Stop name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="KLed" name="kledStopValidated">
     <property name="state">
      <enum>KLed::Off</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serviceProvider</tabstop>
  <tabstop>city</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>updateAutomatically</tabstop>
  <tabstop>updateTimeout</tabstop>
  <tabstop>showRemainingMinutes</tabstop>
  <tabstop>showDepartureTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
